src/main/java/complete_task_use_case/CompleteTaskDataAccess.java

```diff
@@ -0,0 +1,44 @@
+package complete_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessTask;
+import data_access.WeekDataAccess;
+
+import java.util.HashMap;
+
+/**
+ * Task Completion Data Access Class. Implements methods specified by the DsGateway:
+ * Mutating a task and boolean reversing the completed attribute
+ * @author Fardin Faruk
+ */
+
+public class CompleteTaskDataAccess extends WeekDataAccess implements CompleteTaskDsGateway {
+    /**
+     *
```

Comment:
Please add a brief description of what this method does.

Reply:
done